cluster/src/main/java/com/hartwig/pipeline/CommandLineOptions.java

```diff
@@ -373,7 +374,7 @@ public static Arguments from(String[] args) throws ParseException {
 
     public static Context analysisContext(final CommandLine commandLine, final Arguments defaults) {
```

Comment:
Does seem like we're being inconsistent here, is it still an analysis context (ie pipeline analysis context)? Or is it a pipeline context?